### DISCUSSION TITLE
Will there be a Mac Mx support (clangd)

### DISCUSSION THREAD

**Discussion by Rekwass** [mason-org/mason.nvim#227] [Q&A] [answered]
Hello, when I install clangd, it installs the x86_64 version. Is this normal ?

masOS 12.5, Apple M

<img width="761" alt="Screenshot 2022-08-02 at 14 40 14" src="https://user-images.githubusercontent.com/72019227/182378950-e2139a07-1c49-4425-bf97-0b39587f7156.png">

**Comment by williamboman** [comment, marked as answer]
Hello! The clangd package will install using the prebuilt binaries provided in `clangd/clangd`, there are none for Mac ARM yet. Is this causing any issues or is it just a matter of `amd64 != arm64`?

**Reply by Rekwass** [reply]
I don't have any issues for now. It anything is not working, I will let you know.
